File: do_some_exercises/exercises_test.js
var test = require('tape'); 
import { palindrome } from './exercises';


test('palindrome test', function(t) {
    t.plan(3);
    t.equal(palindrome("abc"), false);
    t.equal(palindrome("racecar"), true);
    t.equal(palindrome("z"), true);
});
 

File: do_some_exercises/exercises_test.cpp
#include "gtest/gtest.h"
#include "exercises.h"
#include "string"

TEST(ReverseStrTest, Blahblah) {
	std::string x = "blahblah";
    std::string reversedx ="halbhalb";
	reverseStr(x);
	EXPECT_TRUE(x == reversedx);
}

TEST(ReverseStrTest, Racecar) {
	std::string x = "racecar";
    std::string reversedx ="racecar";
	reverseStr(x);
	EXPECT_TRUE(x == reversedx);
}
 


int main(int argc, char **argv) {
	testing::InitGoogleTest(&argc, argv);

	return RUN_ALL_TESTS();
}
